test/framework/awsiam.go

```diff
@@ -0,0 +1,90 @@
+package framework
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	eksdv1alpha1 "github.com/aws/eks-distro-build-tooling/release/api/v1alpha1"
+
+	"github.com/aws/eks-anywhere/internal/pkg/api"
+	"github.com/aws/eks-anywhere/internal/pkg/awsiam"
+	"github.com/aws/eks-anywhere/pkg/cluster"
+	"github.com/aws/eks-anywhere/pkg/version"
+)
+
+const (
+	AWSIamRoleArn = "T_AWS_IAM_ROLE_ARN"
+	AWSIamUserArn = "T_AWS_IAM_USER_ARN"
```

Comment:
What's this for?

Reply:
So for creating a role mapping for iam-auth I get a RoleArn and a UserArn as an Env variable.
I am currently using only [RoleArn](https://github.com/aws/eks-anywhere/pull/556/files#diff-d9f3a86c63a87fd8ecc0a90afd5e7f4c91e6a71fb2c1103582dc3a2986d8f07aR41) but if someone wanted to test locally they could write a tests using their IAM user Arn.

Reply:
I see, but is this var being used anywhere?

Reply:
`AWSIamUserArn` is currently not being used.
But it can be used similar to how `AWSIamRoleArn` while setting options for the test.
I could remove it if you feel it’s something that can be added later?

Reply:
Removed const `AWSIamUserArn` to add if required later.